### PR TITLE
Adding extra contents for MacOS

### PR DESCRIPTION
**Pull request by wuzhong-papermoon**
### Description
Adding instructions and tips for running a node on MacOS.

### Checklist

- [ ] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [✅] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [✅] If pages have been moved around, I have run the `move-pages.py` script to move the pages and update the image paths on the chinese repo
    - [ ] After the script has been run, I have created an additional PR in `moonbeam-docs-cn`
- [✅] If images have been added, I have run the `compress-images.py` script to compress the images.
- [✅] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables
- [✅] If this page requires a disclaimer, I have added one

### After Translation Requirements
- [ ] No additional PRs are required after the translations are done

### CODE REVIEW

File: builders/get-started/networks/moonbeam-dev.md [hunk 1]
```diff
@@ -90,6 +95,9 @@ To build the binary file, you can take the following steps:
     cd moonbeam
     ```
 
+    !!! note
+        Space in the installation file path will cause an compilation error.
```
Comment (eshaben) [inline builders/get-started/networks/moonbeam-dev.md:99]: I am a little confused by this comment, do you mean if the user clones the repository in a directory that has a space in it? so the path is something like this?:

```
user/my documents/moonbeam
```

where `my documents` has a space in it?

File: builders/get-started/networks/moonbeam-dev.md [hunk 2]
```diff
@@ -111,6 +119,13 @@ To build the binary file, you can take the following steps:
         apt install clang protobuf-compiler libprotobuf-dev -y 
         ```
 
+        For MacOS users, these dependencies can be installed via homebrew:
+
+        ```bash
+        brew install llvm
+        brew install protobuf
+        ```
```
Comment (eshaben) [inline builders/get-started/networks/moonbeam-dev.md:127]: I'm curious, did you run into an error that required you to install these on your mac? If so, do you recall what the exact error was?

File: builders/get-started/networks/moonbeam-dev.md [hunk 3]
```diff
@@ -62,9 +62,14 @@ Using Docker enables you to spin up a node in a matter of seconds. Once you have
         --dev --rpc-external
         ```
 
-    If successful, you should see an output showing an idle state waiting for blocks to be authored:
+!!! note
+    On macOS with silicon chips, Docker images may perform poorly. To improve performance, try [spinning up a Node with a Binary File](#getting-started-with-the-binary-file).
```
Comment (eshaben) [inline builders/get-started/networks/moonbeam-dev.md:66]:
```suggestion
    On MacOS with silicon chips, Docker images may perform poorly. To improve performance, try [spinning up a Node with a Binary File](#getting-started-with-the-binary-file).
```

**Review by eshaben — COMMENTED** [review]
Woo! First PR! 🎉 

Just had a couple questions 🙂

File: builders/get-started/networks/moonbeam-dev.md (hunk 1 above)
Comment (wuzhong-papermoon) [reply builders/get-started/networks/moonbeam-dev.md:99]: This is correct!

Comment (eshaben) [inline builders/get-started/networks/moonbeam-dev.md:99]:
```suggestion
        Space in the installation file path will cause a compilation error.
```

File: builders/get-started/networks/moonbeam-dev.md [hunk 4]
```diff
@@ -111,6 +119,13 @@ To build the binary file, you can take the following steps:
         apt install clang protobuf-compiler libprotobuf-dev -y 
         ```
 
+        For MacOS users, these dependencies can be installed via homebrew:
```
Comment (eshaben) [inline builders/get-started/networks/moonbeam-dev.md:122]:
```suggestion
        For MacOS users, these dependencies can be installed via Homebrew:
```

**Review by eshaben — CHANGES_REQUESTED** [review]
Cool, just a couple minor suggestions! You can add all of the suggestions to a batch from the "Files changed" tab and commit them at once. I find that to be the easiest way, but you can also make the changes locally if you prefer!

**Comment by wuzhong-papermoon** [comment]
Done! Thanks for reviewing it! 
Thanks for the tip on using the "Files changed" tab. I didn't know we could do it that way, it's super helpful!

**Review by eshaben — APPROVED** [review]
LGTM

**Pull request merged**